router/expressRouter.js

```diff
@@ -0,0 +1,8 @@
+import express from "express";
+const router = express.Router();
+
+router.route("/readUser").get();
+
+router.route("/createUser").post();
```

Comment:
Already a nice step. Just the implementation of the post() route is missing. But I stay tuned :100: